xapian-core/tests/harness/backendmanager.h

```diff
@@ -120,6 +120,13 @@ class BackendManager {
     /// Get a writable database instance.
     virtual Xapian::WritableDatabase get_writable_database(const std::string & name, const std::string & file);
 
+    /** Get alternate writable database for backends which
+     * don't support writing directly.
+     */
+    virtual Xapian::WritableDatabase get_alt_writable_database
+					(const std::string & name,
+					 const std::string & file);
```

Comment:
I think it's better to wrap after the return type if wrapping the parameter list isn't sufficient (since we wrap after the return type in out-of-line function definitions), like so:

```
    virtual Xapian::WritableDatabase
    get_alt_writable_database(const std::string& name,
			      const std::string& file);
```